ostd/src/arch/x86/mod.rs

```diff
@@ -114,7 +114,8 @@ fn enable_common_cpu_features() {
     }
 
     let mut xcr0 = x86_64::registers::xcontrol::XCr0::read();
-    xcr0 |= XCr0Flags::AVX | XCr0Flags::SSE;
+    let avx512 = XCr0Flags::OPMASK | XCr0Flags::ZMM_HI256 | XCr0Flags::HI16_ZMM;
+    xcr0 |= XCr0Flags::AVX | XCr0Flags::SSE | avx512;
```

Comment:
`FpRegs::save` and `FpRegs::restore` methods are implemented, but never used. In other words, the AVX-related CPU registers are never saved and restored. Without doing so, we won't have correct AVX states after context switching between user-space threads. So enabling AVX is more than just setting a few bits in CR0.

Reply:
Got it, thanks!
I'm not very familiar with the upcoming effort here, I guess in this PR we had three options:
1. Fix this correctness issue in this PR ASAP, before that the FIO tool won't be worked;
2. Add a `FIXME` here and I fix the problem ASAP in a later PR;
3. Add `--disable-optimizations` to the `configure` stage in compiling FIO (and AVX-512 related instructions won't be used), which requires changing dockerfile and another bump version. And I'll fix this in a more appropriate time.

Which do you prefer?